packages/salesforcedx-vscode-apex/src/oas/generationStrategy/prompts.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export const prompts = {
```

Comment:
@CristiCanizales I believe there will be merge conflicts between this new file and the one is also creating in his PR. I see the need for these prompts to be categorized based on each generator.

I suggest claiming a top-level key for those prompts belonging to the whole class generator.